source/js/init.js

```diff
@@ -0,0 +1,125 @@
+let recipes = {};
```

Comment:
Codacy has a fix for the issue: ['recipes' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-fa21-group7/cse110-fa21-group7/pullRequest?prid=8446960)
```suggestion
const recipes = {};
```